svelte/src/components/markdown/markdown.svelte

```diff
@@ -24,7 +23,15 @@
     link: Link
   };
 
-  $: html = source.type === "rst" ? rst2html(source.data) : "";
+  $: html = source.type === "rst" ? rst2html(source.data) : source.data;
+
+  // TODO: support gitlab too
+  const changeSrcIfNeeded = (element: HTMLElement | Element) => {
+    const src = element.getAttribute("src");
+    if (src?.startsWith("/")) {
```

Comment:
I'm not sure this single condition will cover all the bases for relative URLs. I had found most examples of a relative path in the Openssl, lefthook, and xcodegen libraries. Among those three you get srcs like:

```
assets/imgs/foo.png
/assets/imgs/foo.png
./assets/imgs/foo.png
```

I had begun working on a remedy to this issue yesterday before the s3 solution was mentioned/implemented, and a regular expression covered the bases for relative paths we'd run in to in READMEs. Something like:
`const isRelativePath = src?.match(/^[^\/]+\/[^\/].*$|^\/[^\/].*$/i)`

Reply:
Note: This regex could probably be simplified, as it covers conditions we won't likely run in to as well

Reply:
@ryanml ill add that thank you, i missed out on that hehehe